extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.Optimizely.js

```diff
@@ -0,0 +1,29 @@
+/* global toolbarABTestVariantNumber:false */
+/*!
+ * Utility functions for running Optimizely A/B tests with VisualEditor
+ */
+
+/**
+ * @returns {number} The A/B test variant number for the Toolbar A/B test
+ */
+ve.init.wikia.getToolbarABTestVariantNumber = function () {
+	// toolbarABTestVariantNumber is set for variant #1 for all Optimizely toolbar experiments
+	if ( typeof ( toolbarABTestVariantNumber ) === 'undefined' ) {
```

Comment:
There are still only two UI variants, so each of the language Optimizely tests set toolbarABTestVariantNumber to 1 for Variant #1, which triggers the new toolbar to show.